### ISSUE TITLE
Missing Music Events in chinese2.c (Crystal Maze)

### ISSUE THREAD

**Issue by slavexe** [DurtyFree/gta-v-data-dumps#45]
I noticed that most music events from chinese2.c are missing in the music event data dump. This script in particular appears to be using an extra function called ```func_15``` to play music events rather than directly calling ```AUDIO::TRIGGER_MUSIC_EVENT```. For example: ```func_15("CHN2_STOP_TRACK", 0, func_173());``` There may be other scripts that do this, but this is the only one I have found so far.

**Comment by DurtyFree** [comment]
Interesting, but I guess its hard to really track down these and keeping track of them. As of right now its really just reading all uses of the related natives.

Missing things like this occur in many other dumps too as they often use functions and input the string we want as a arg. I think the only way to efficiently dump these would be some logical way of parsing the script and really understanding whats going on. Something I might work on in the future. 
Until then I will keep this open

If some one else has a smart idea of automatically tracking such cases, feel free to discuss here!

**Comment by slavexe** [comment]
Yeah I can't think of a good way of fixing this, but I also found exile1.c handles music events in this way too.
